src/test/java/edu/usf/cutr/gtfsrtvalidator/test/feeds/TripUpdateFeedTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package edu.usf.cutr.gtfsrtvalidator.test.feeds;
+
+import edu.usf.cutr.gtfsrtvalidator.FeedMessageTest;
+import edu.usf.cutr.gtfsrtvalidator.validation.entity.CheckTripId;
+import edu.usf.cutr.gtfsrtvalidator.validation.entity.StopTimeSequanceValidator;
+import edu.usf.cutr.gtfsrtvalidator.validation.entity.VehicleIdValidator;
+import edu.usf.cutr.gtfsrtvalidator.validation.gtfs.StopLocationTypeValidator;
+import org.junit.Test;
+
+
+public class TripUpdateFeedTest extends FeedMessageTest{
+    
+    public TripUpdateFeedTest() throws Exception{}
+    
+    @Test
+    public void testTripIdValidation(){
+        
+        CheckTripId tripIdValidator = new CheckTripId();
+        
+        // feedMessage already have a valid trip_id set in InitRequiredFeedFields() method
+        errors = tripIdValidator.validate(gtfsData, feedMessage.build());
+        assertEquals(0, errors.getOccurrenceList().size());
+        
+        tripDescriptor.setTripId("100");
+        tripUpdate.setTrip(tripDescriptor);
+        feedEntity.setTripUpdate(tripUpdate);
+        feedMessage.setEntity(0, feedEntity);
+        
+        errors = tripIdValidator.validate(gtfsData, feedMessage.build());
+        assertEquals(1, errors.getOccurrenceList().size());
+        
+        clearAndInitRequiredFeedFields();
+    }
+    
+    @Test
+    public void testStopSequenceValidation() {
+        StopTimeSequanceValidator stopSequenceValidator = new StopTimeSequanceValidator();
+        
+        //ordered stop sequence 1, 5
+        stopTimeUpdate.setStopSequence(1);
+        tripUpdate.addStopTimeUpdate(stopTimeUpdate);
+        stopTimeUpdate.setStopSequence(5);
```

Comment:
please correct me, if this is not what you meant in the comment :)
each time **addStopTimeUpdate** will add a new `stopTimeUpdate` to the list of `stopTimeUpdate` in `tripUpdate`. So for ordered stopSequence validation, we have 2  `stopTimeUpdates` in `tripUpdate` and for unordered stopSequence validation, we have 3 `stopTimeUpdates` in `tripUpdate`

I also now added assertion tests for the number of `stopTimeUpdates` in `tripUpdate` to make sure there are 2 for ordered stopSequence validation and 3 for unordered stopSequence validation.